### DISCUSSION TITLE
Suggestions :)

### DISCUSSION THREAD

**Discussion by nikeymikey** [pearlxcore/PS4PKGTool#43] [General]
Hi, great, very useful app you have made! I have a couple of suggestions for new features, would it be possible to add a way to organise the actual pkg's into folders. So for example if i wanted to put all VR games together in one folder, i could select just the VR games and the app would move them all into a selected folder.
This could also be useful to organise dlc from one game all into one folder. Or even game, patch and dlc all together into one folder. 

Second idea is when adding a custom naming scheme for renaming one or all pkg's would it be possible to save the custom naming scheme for future use? 

Keep up the great work and thanks for making such a useful app.

**Comment by nikeymikey** [comment]
One other thing I thought of.... Is it possible to add PSVR as a tag for naming?

**Comment by JonJonSalchichon** [comment]
Hi! Great work. Love the tool.
One idea here (my 5 cents) allowing to add a forlder from a network drive.
I personally have my games in a NAS drive (and not in my PC or in a USB disk) and it would be very nice to be able to retrieve the PKG file directly from there.
Thanks and kind regards,
